app/routes/users/components/UserProfile/index.tsx

```diff
@@ -157,12 +157,17 @@ const UserProfile = () => {
       title={user.fullName}
       actionButtons={
         showSettings && (
-          <Icon
-            iconNode={<SettingsIcon />}
-            size={22}
-            className={styles.settingsIcon}
-            to={`/users/${user.username}/settings/profile`}
-          />
+          <LinkButton
+            className={styles.settingsButton}
+            href={`/users/${user.username}/settings/profile`}
+          >
+            <Icon
+              iconNode={<SettingsIcon />}
+              size={22}
```

Comment:
`22` is a bit large when it is inside a button - perhaps we could do something like `19` (like we do for most other buttons with icons)